core/core/services/contact.py

```diff
@@ -50,3 +53,9 @@ def get_contact(payload: ContactPayload) -> Contact:
         return record
     else:
         return None
+
+
+@response()
+@write_data(entity=Entities.CONTACTS)
+def delete_contact(payload: ContactPayload) -> bool:
+    return database.delete(entity=Entities.CONTACTS, key=payload.name)
```

Comment:
it should be 
return database.select(entity=Entities.CONTACTS, key=payload.name)